src/main/java/martin/exceptions/InvalidCommandException.java

```diff
@@ -0,0 +1,12 @@
+package martin.exceptions;
+
+/**
+ * Represents an exception when command is invalid.
+ *
+ * @param message Error message to be printed out.
+ */
+public class InvalidCommandException extends MartinException {
```

Comment:
I feel maybe don't so many classes for each exceptions since all your class methods are implementing same line of code super(message).